### DISCUSSION TITLE
[Question] : Beeware

### DISCUSSION THREAD

**Discussion by jhay06** [beeware/toga#2289] [Q&A]
Just want to check on beeware , 
we all know that there is also kivy alternative but some packages and bugs still exist , so i'm planning to migrate to other python framework for cross platform .

i would like to know below.
1. does .py file when compiling will convert to pyc/pyo/etc. ? 
2. where does pypi packages list .? 
3. does beeware supports svg ? 
4. does beeware use native interface or it uses canvas to draw gui? 

Thank you

**Comment by freakboy3742** [comment]
1. I'm not completely sure I understand what you're asking - but BeeWare runs Python code as normal python; There's no compilation step. The default Briefcase deployment will also disable the creation of bytecode files (i.e., pyc files).
2. You can use any pure Python package on PyPI, and any binary package that exists for your target platform. For desktop platforms, this usually isn't a problem; but for iOS and Android, the selection is somewhat limited. See [this](https://anaconda.org/beeware/) for the iOS supported list, and [this](https://chaquo.com/pypi-13.1/) for the Android list.
3. Depends what you mean by "support". There's no native mechanism to "load SVG"; but the web canvas will support as much SVG as the underlying platform's web view supports.
4. BeeWare is 100% native widgets. A button in Toga is an NSButton on macOS, a Winforms button on Windows, a GTK Button on Linux, and so on.
